app/src/androidTest/java/com/duckduckgo/app/TestExtensions.kt

```diff
@@ -37,7 +39,7 @@ fun <T> LiveData<T>.blockingObserve(): T? {
 }
 
 fun getApp(): TestApplication {
-    return InstrumentationRegistry.getTargetContext().applicationContext as TestApplication
+    return InstrumentationRegistry.getInstrumentation().targetContext.applicationContext as TestApplication
```

Comment:
`InstrumentationRegistry.getInstrumentation().targetContext`

This is the new way to get hold of a `Context` for your target app during unit testing - so you'll see this change a lot in this PR. the old way is deprecated now - think i got them all.